Analysis/Tasks/PWGCF/correlations.cxx

```diff
@@ -271,7 +271,9 @@ struct CorrelationTask {
     LOGF(info, "Tracks for collision: %d | Vertex: %.1f | INT7: %d | V0M: %.1f", tracks.size(), collision.posZ(), collision.sel7(), collision.centV0M());
 
     if (std::abs(collision.posZ()) > cfgCutVertex) {
-      LOGF(warning, "Unexpected: Vertex %f outside of cut %f", collision.posZ(), cfgCutVertex);
+      std::stringstream tmp;
+      tmp << cfgCutVertex;
+      LOGF(warning, "Unexpected: Vertex %f outside of cut %f", collision.posZ(), tmp.str());
```

Comment:
Does this actually work? Shouldn't the second one be %s?

Reply:
I would assume it should be %s, but it compiled for me as is.
Actually I think `(float)cfgCutVertex` would be even better.

Reply:
agreed about `(float)cfgCutVertex`.